packages/dd-trace/src/appsec/blocking.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 
 const fs = require('fs')
+let templateLoaded = false
 let templateHtml, templateJson
-function block (req, res, topSpan, abortController) {
+function block ({ req, res, topSpan, abortController }) {
```

Comment:
Also maybe the actual name should be `rootSpan` and not `topSpan` since we've been using the former in most other places in the SDK

Reply:
At first there were also the status code and body content passed to the function but since we decided to remove them I agree that the object parameter might not be needed.